include/oneapi/tbb/detail/_export.h

```diff
@@ -18,27 +18,29 @@
 #define __TBB_detail__export_H
 
 #if defined(__MINGW32__)
-    #define _EXPORT __declspec(dllexport)
```

Comment:
BTW can [GENERATE_EXPORT_HEADER](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html)  be used instead of hand written ?

Reply:
I don't know. Need experiments.
But I think this is out of scope for this PR.

Reply:
> BTW can [GENERATE_EXPORT_HEADER](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) be used instead of hand written file?

As far as we can see TBB can be built using bazel and other tools. So, having strict dependency on cmake generation of public header file is not a good idea.